movies/movie_api_utils.py

```diff
@@ -85,3 +85,23 @@ def _get(self, url, **kwargs) -> dict[str, Any]:
             raise MovieApiException("Request to the movie database timed out.")
         except requests.exceptions.RequestException:
             raise MovieApiException("An error occurred while processing your request.")
+
+    def convert_date_string_into_object(self, movies, *, filter: str = None):
```

Comment:
The docstring here could benefit from an example of how the filter would be used.
Is it to search for a certain movie? date?